### DISCUSSION TITLE
Add endorsement info

### DISCUSSION THREAD

**Discussion by mattpolzin** [SwiftPackageIndex/SwiftPackageIndex-Server#1352] [Ideas]
Not really sure how this manifests, but the idea I had was to provide an indication at a glance that a library is endorsed by one of the bodies predetermined to be influential. For example, a package being endorsed by the Swift Server Work Group would be one of the factors I considered when choosing what library to use because the SSWG has certain standards and goals that give me confidence both in the quality of a library and in the interoperability gained by choosing to use that library. 

I imagine this being a curation feature. The list of SSWG libraries would be manually maintained rather than something in the individual package indicating membership. I’m not sure if there are other applicable groups for which endorsement would also be useful information right now but at least with time there might be. 

*personal note: I am a library author with a library that is incubating with the SSWG so I am naturally biased but I am also confident that this feature would help me when I am on the search for other libraries.

**Comment by finestructure** [comment]
I think this would be a good thing to add over in the metadata thread: #202

**Comment by daveverwer** [comment]
Hey Matt! Thank you so much for this.

I've been doing some work on this today and started a new thread to clarify and consolidate #202. I included your suggestion as part of that consolidated thread and linked to this issue for context. The new issue is #435.

Thank you for the suggestion and please feel free to contribute in that thread. I'll close this issue just to keep all discussion in one place.

**Comment by daveverwer** [comment]
I took this to mean that the packages would self declare this, but [Max is right](https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/issues/435#issuecomment-647151597) that it's such a small list it's something we could just do manually.

Re-opening this issue to track that.
